skimage/filters/_gaussian.py

```diff
@@ -96,6 +98,11 @@ def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
     >>> filtered_img = gaussian(image, sigma=1, multichannel=True)
 
     """
+    if preserve_range is None:
+        msg = ('The default value of preserve_range will change '
+               'to True in version 1.0')
+        warn(msg, DeprecationWarning)
+        preserve_range = False
```

Comment:
I think the `change_default_value` decorator utility @rfezzani created can be used to do this. To use it add the following import near the top of the file:
```Python
from .._shared.utils import change_default_value
```

Then rather than changing the value to None or raising the warning here, all you need is a 1 line decorator above the function

```Python
@change_default_value('preserve_range', new_value=True, changed_version='1.0')
```

Reply:
@grlee77 Thanks , that looks good. You may check now

By the way I would like to suggest to add this decorator to [Deprecation Cycle Documentation](https://scikit-image.org/docs/dev/contribute.html#deprecation-cycle)